base-docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "3.2"
+version: "2.4"
```

Comment:
will the docker-compose with multiple .yml files having different version work properly?
in general, I propose to have the same version everywhere

Reply:
yes this is still to correct - it will be corrected in all the subrepos (or maybe i did already) but didn't update it here yet

Reply:
afaik it already has 3.4 in all of the other repos

Reply:
and also the answer to my question: no, it won't

Reply:
it does in the repo, but the PRs are addressing that (because as you said, it wouldn't work)